integration/testdata/helm.json.golden

```diff
@@ -268,7 +268,7 @@
       "Class": "config",
       "Type": "helm",
       "MisconfSummary": {
-        "Successes": 34,
+        "Successes": 78,
```

Comment:
Did you add a lot of checks for Kubernetes?

Reply:
Haha, yes! @josedonizetti added lots of CIS Benchmark checks in https://github.com/aquasecurity/defsec/pull/720 and https://github.com/aquasecurity/defsec/pull/721 :)

Reply:
Oh, cool👍 I just wanted to confirm that a number of `Successes` didn't jump by mistake. Thanks for confirming.